source/_integrations/fan.mqtt.markdown

```diff
@@ -270,6 +270,8 @@ In this section you find some real-life examples of how to use this fan.
 The example below shows a full configuration for a MQTT fan using percentage and preset modes.
 There are 10 speeds within the speed range, so  `percentage_step` = 100 / 10 steps = 10.0 %.
 
+{% raw %}
```

Comment:
There is no liquid syntax in the YAML example below, is there a reason this was added?

Reply:
It was added to give an example with command templates, that might be referred elsewere. The only thing missing was a correct index.
About `liquid syntax`, my knowlegde lacks a bit. What should be different to get a `liquid syntax`?
If your statement is that this example should not be in here since we have https://www.home-assistant.io/docs/configuration/templating/ then, let me know. I could remove the example too.

Reply:
Jekyll (the static site generate) uses liquid syntax, which might be triggered by Jinja2 syntax (as they share similarities). This when using Jinja2 (e.g., templates) we need to use the raw tags. However, this example does not have templating, thus I don't see why the raw is needed here.

Reply:
Ah, without the raw tag, the example below is not showing correct. The first example could do without raw tags. I added them for consistency. I will remove those.